src/components/inputs/PasswordAdornment/PasswordAdornment.tsx

```diff
@@ -0,0 +1,21 @@
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import VisibilityOffIcon from '@mui/icons-material/VisibilityOff';
+import {IconButton, InputAdornment} from '@mui/material';
+
+interface PasswordAdornmentProps {
+  showPassword: boolean;
+  handleClickShowPassword: () => void;
+  handleMouseDownPassword: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+const PasswordAdornment: React.FC<PasswordAdornmentProps> = ({showPassword, handleClickShowPassword, handleMouseDownPassword}) => {
```

Comment:
Why `React.FC` not `FC`?

Reply:
A habbit. Can type FC. Even better.

Reply:
> Why `React.FC` not `FC`?

PS: but if I use FC I need to import import {FC} from 'react'. With React.FC no need for import.